lib/utils/opType.js

```diff
@@ -0,0 +1,35 @@
+const stripAnsiChars = require('./stringUtils').stripAnsiChars;
+
+// Our chains can be either "A" assert or "E" - evaluation.
```

Comment:
What about 'Query' lines ?

Reply:
Anything that does not throw an exception when the server responds with queryFailed is considered evaluation, everything else is considered assertion. I think this covers query lines?